pandas-stubs/core/frame.pyi

```diff
@@ -166,7 +166,9 @@ class _LocIndexerFrame(_LocIndexer):
     @overload
     def __getitem__(
         self,
-        idx: tuple[int | StrLike | tuple[ScalarT, ...], int | StrLike],
+        idx: tuple[
+            int | StrLike | tuple[ScalarT, ...], int | StrLike | tuple[ScalarT, ...]
```

Comment:
Would `Scalar` also work here instead of `ScalarT`?

Reply:
> Would `Scalar` also work here instead of `ScalarT`?

Yes, seems so.  Fixed in commit https://github.com/pandas-dev/pandas-stubs/pull/494/commits/3417725f9e174a36595017559c1f3e1c2419e807